### PR TITLE
Build lambda in zip

### PR DESCRIPTION
**Pull request by winston0410**
Zip build and deploy to serverless

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -76,12 +77,19 @@
     "@storybook/manager-webpack5": "6.3.12",
     "@storybook/preset-scss": "^1.0.3",
     "@storybook/react": "6.3.12",
+    "@swc/core": "^1.2.133",
+    "@testing-library/jest-dom": "^4.2.4",
```
Comment (akeamwoodsND) [inline package.json:81]: Can any of these new packages be moved to devDependencies?

File: package.json [hunk 2]
```diff
@@ -76,12 +77,19 @@
...
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "@zeit/next-less": "^1.0.1",
+    "@zeit/next-sass": "^1.0.1",
+    "@zeit/next-stylus": "^1.0.1",
     "babel-loader": "^8.2.2",
     "babel-preset-react-app": "^10.0.0",
     "chromatic": "^6.0.4",
+    "critters": "^0.0.16",
```
Comment (akeamwoodsND) [inline package.json:90]: Where is this being used?

Comment (akeamwoodsND) [reply package.json:90]: Are these all being used?

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (winston0410) [reply package.json:90]: All these are used for fixing the dependencies for bundling Next.js, which is the dependencies they have left out. And yes these should be moved to devDependencies

File: package.json (hunk 1 above)
Comment (winston0410) [reply package.json:81]: I thinks these dependencies are in devDependencies already

**Pull request merged**